util/cmd.py

```diff
@@ -257,4 +260,28 @@ def check_input(condition, error_msg):
     if not condition:
         raise BadInputError(error_msg)
 
+def parse_cmd(module, cmd, args):
+    """Parse arguments `args` to command `cmd` from module `module`."""
+    if isinstance(module, str):
+        module = importlib.import_module(module)
+    assert inspect.ismodule(module)
+    parser_fn = dict(getattr(module, '__commands__'))[cmd]
+    return parser_fn(argparse.ArgumentParser()).parse_args(list(map(str, args)))
+
+CmdRunInfo = collections.namedtuple('CmdRunInfo', ['result', 'args_parsed'])
+
+def run_cmd(module, cmd, args):
+    """Run command after parsing its arguments with the command's parser.
     
+    Args:
```

Comment:
What led to these changes, or what do they fix?

Reply:
These are useful for testing, if you want to run a command but also see the results of parsing its arguments, e.g. https://github.com/broadinstitute/viral-ngs/pull/854/files#diff-771ac3c0ea113ca9a6634523f4fb6893R273
Also calling run_cmd() is clearer than the current pattern of calling the parser manually then calling its func_main .